README.md

```diff
@@ -4,7 +4,52 @@ The Ping CLI is a unified command line interface for configuring and managing Pi
 
 ## Install
 
-<!-- TODO -->
+#### macOS/Linux - Homebrew
+
+Use PingIdentity's Homebrew tap to install Ping CLI
+
+```text
+brew install pingidentity/tap/pingcli
+
+or
+
+brew tap pingidentity/tap
+brew install pingcli
+```
+
+#### Manual Installation - macOS/Linux
+
+See [the latest GitHub release](https://github.com/pingidentity/pingcli/releases/latest) for binary downloads and SHA256 checksum files.
+
+OR
+
+Use the following single-line command to install Ping CLI directly.
+
+```text
+RELEASE_VERSION=$(basename $(curl -Ls -o /dev/null -w %{url_effective} https://github.com/pingidentity/pingcli/releases/latest)); \
+OS_NAME=$(uname -s); \
+HARDWARE_PLATFORM=$(uname -m | sed s/aarch64/arm64/ | sed s/x86_64/amd64/); \
+curl --location --silent --output pingcli.tar.gz --request GET \
+"https://github.com/pingidentity/pingcli/releases/download/${RELEASE_VERSION}/pingcli_${RELEASE_VERSION#v}_${OS_NAME}_${HARDWARE_PLATFORM}.tar.gz"; \
```

Comment:
something weird about this line when copying. tried various ways of copying and each one adds an extra escape near the end that breaks the curl.

```
RELEASE_VERSION=$(basename $(curl -Ls -o /dev/null -w %{url_effective} https://github.com/pingidentity/pingcli/releases/latest)); \
OS_NAME=$(uname -s); \
HARDWARE_PLATFORM=$(uname -m | sed s/aarch64/arm64/ | sed s/x86_64/amd64/); \
curl --location --silent --output pingcli.tar.gz --request GET \
"https://github.com/pingidentity/pingcli/releases/download/${RELEASE_VERSION}/pingcli_${RELEASE_VERSION#v}_${OS_NAME}_${HARDWARE_PLATFORM}.tar.gz"\; \
tar -zxf pingcli.tar.gz pingcli; \
rm -f pingcli.tar.gz
```
this is from my paste

Reply:
Resolved by breaking out the URL formation into its own line. This was an issue with oh-my-zsh MAGIC_FUNCTIONS.